<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExportSvgDialog</class>
 <widget class="QDialog" name="ExportSvgDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>250</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Export SVG Options</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupFillStroke">
     <property name="title">
      <string>Fill and Stroke</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayoutFillStroke">
      <item>
       <widget class="QCheckBox" name="checkBoxEnableFill">
        <property name="toolTip">
         <string>Enable filling of exported geometry with a color.</string>
        </property>
        <property name="text">
         <string>Fill Geometry</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayoutFillColor">
        <item>
         <widget class="QLabel" name="labelFillColor">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::Box</enum>
          </property>
          <property name="lineWidth">
           <number>2</number>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolButtonFillColorReset">
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset theme="chokusen-undo">
           <normaloff>.</normaloff>.</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolButtonFillColor">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBoxEnableStroke">
        <property name="toolTip">
         <string>Enable outline stroke for exported geometry.</string>
        </property>
        <property name="text">
         <string>Stroke Outline</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayoutStrokeColor">
        <item>
         <widget class="QLabel" name="labelStrokeColor">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::Box</enum>
          </property>
          <property name="lineWidth">
           <number>2</number>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolButtonStrokeColorReset">
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset theme="chokusen-undo">
           <normaloff>.</normaloff>.</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolButtonStrokeColor">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayoutStrokeWidth">
        <item>
         <widget class="QLabel" name="labelStrokeWidth">
          <property name="text">
           <string>Stroke Width:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="doubleSpinBoxStrokeWidth">
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="suffix">
           <string> mm</string>
          </property>
          <property name="decimals">
           <number>2</number>
          </property>
          <property name="minimum">
           <double>0.01</double>
          </property>
          <property name="maximum">
           <double>10.00</double>
          </property>
          <property name="singleStep">
           <double>0.10</double>
          </property>
          <property name="value">
           <double>0.35</double>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolButtonStrokeWidthReset">
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset theme="chokusen-undo">
           <normaloff>.</normaloff>.</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutDialogButtons">
     <item>
      <spacer name="horizontalSpacerDialogButtons">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonOk">
       <property name="text">
        <string>OK</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>checkBoxEnableFill</tabstop>
  <tabstop>toolButtonFillColorReset</tabstop>
  <tabstop>toolButtonFillColor</tabstop>
  <tabstop>checkBoxEnableStroke</tabstop>
  <tabstop>toolButtonStrokeColorReset</tabstop>
  <tabstop>toolButtonStrokeColor</tabstop>
  <tabstop>doubleSpinBoxStrokeWidth</tabstop>
  <tabstop>toolButtonStrokeWidthReset</tabstop>
  <tabstop>pushButtonCancel</tabstop>
  <tabstop>pushButtonOk</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonOk</sender>
   <signal>clicked()</signal>
   <receiver>ExportSvgDialog</receiver>
   <slot>accept()</slot>
  </connection>
  <connection>
   <sender>pushButtonCancel</sender>
   <signal>clicked()</signal>
   <receiver>ExportSvgDialog</receiver>
   <slot>reject()</slot>
  </connection>
 </connections>
</ui> 
